libs/libmbim/Makefile

```diff
@@ -11,24 +11,28 @@ PKG_NAME:=libmbim
 PKG_VERSION:=1.26.2
 PKG_RELEASE:=$(AUTORELEASE)
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=https://www.freedesktop.org/software/libmbim
-PKG_HASH:=10c77bf5b5eb8c92ba80e9b519923ad9b898362bc8e1928e2bc9a17eeba649af
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.freedesktop.org/mobile-broadband/libmbim.git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
```

Comment:
The interesting part here is that you can set this to 1.26.2 and remove PKG_VERSION as the latter is set from the former.

Reply:
Thanks. Should I replace PKG_VERSION with PKG_SOURCE_VERSION and remove line 16? I relied on the Makefile of other packages (for example libqrtr-glib)

Reply:
I'd remove PKG_VERSION and put it in PKG_SOURCE_VERSION. Keep in mind the latter is needed.

Reply:
Ok, done.